install/autodetect.go

```diff
@@ -98,10 +98,7 @@ func (k *K8sInstaller) autodetectAndValidate(ctx context.Context) error {
 		}
 	}
 
-	if k.params.Version == "" {
-		k.Log("ℹ️  Cilium version not set, using default version %q", defaults.Version)
-		k.params.Version = defaults.Version
-	}
+	k.Log("ℹ️  Cilium version not set, using Cilium version %q", defaults.Version)
```

Comment:
I think this statement here not correct anymore. If the user specifies `k.params.Version`, then this will still print `defaults.Version`?

Reply:
ouch :facepalm: my bad, thank you for pointing it out